diff --git a/src/Views/BaseTemplate.php b/src/Views/BaseTemplate.php
index a8a23ac..eec09f3 100644
--- a/src/Views/BaseTemplate.php
+++ b/src/Views/BaseTemplate.php
@@ -33,7 +33,7 @@ class BaseTemplate
                         <a class="nav-link" href="/pizza221/products">Каталог</a>
                         </li>
                         <li class="nav-item">
-                        <a class="nav-link" href="/pizza221/about">О нас</a>
+                        <a class="nav-link" href="/pizza221/about">О компании</a>
                         </li>
                     </ul>
                     </div>
